frontend/src/features/qa/page-list/helpers/issueCounts.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  severityFromMatch,
+  severityFromResourceCounts,
+  type Severity,
+} from "./severity";
+
+import type { ArchivedItemQAPage } from "@/types/qa";
+import { cached } from "@/utils/weakCache";
+
+export const issueCounts = cached((page: ArchivedItemQAPage) => {
+  const severities = [
+    severityFromMatch(page.qa.screenshotMatch),
+    severityFromMatch(page.qa.textMatch),
+    severityFromResourceCounts(
+      page.qa.resourceCounts?.crawlBad,
+      page.qa.resourceCounts?.crawlGood,
+    ),
+    severityFromResourceCounts(
+      page.qa.resourceCounts?.replayBad,
+      page.qa.resourceCounts?.replayGood,
```

Comment:
Similarly, is it guaranteed that we'll have values for props of `resourceCounts`? If not, let's update the type to reflect this.

Reply:
`resourcesCounts` is not guaranteed, but entries within, if it exists, should be. Will update!